### PR TITLE
Replace script_run("zypper") with zypper_call

### PR DESCRIPTION
**Pull request by asdil12**
Ticket: https://progress.opensuse.org/issues/51467

Verification:

File | Verification |
--- | --- |
`tests/console/rabbitmq.pm` | http://artemis.suse.de/tests/1351 |
`tests/virtualization/syscontainer_image_test.pm` | Seems orphaned |
`tests/virtualization/yast_virtualization.pm` | https://openqa.opensuse.org/t957524 |
`tests/x11/chrome.pm` | http://artemis.suse.de/tests/1352#step/chrome/26 |
`tests/x11/libreoffice/libreoffice_mainmenu_components.pm` | Unable to verify because [reason](https://openqa.suse.de/tests/2976407#next_previous) |
`tests/x11/nis_client.pm` | https://openqa.suse.de/tests/2977934 |
`tests/x11/nis_server.pm` | https://openqa.suse.de/tests/2977932#step/nis_server/50 |

### CODE REVIEW

File: tests/console/rabbitmq.pm [hunk 1]
```diff
@@ -17,14 +17,13 @@ use base "consoletest";
...
-use utils 'systemctl';
+use utils;
 
 sub run {
     select_console 'root-console';
-    assert_script_run('zypper -n in rabbitmq-server');
+    zypper_call 'in rabbitmq-server python-pika wget';
     systemctl 'start rabbitmq-server';
     systemctl 'status rabbitmq-server';
-    assert_script_run('zypper -n in python-pika wget');
```
Comment (okurz) [inline tests/console/rabbitmq.pm]: please keep the separate call. "rabbitmq-server" is the service we want to test, "python-pika" and "wget" are merely test requirements hence the later install

**Comment by okurz** [comment]
Looks good, do you plan to replace *all* occurences? Because there still seem to be some

**Comment by asdil12** [comment]
Yes - at least the ones where it is trivial.

**Comment by asmorodskyi** [comment]
LGTM , sadly there is conflicts so I cannot merge . please merge it yourself just after solving conflicts

**Pull request merged**